_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -22,28 +22,28 @@ acknowledgments:
 
 ## Applicability
 
-Any [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is not empty (""), and that is specified on an HTML or SVG element.
+The rule applies to any HTML or SVG element that has any not empty (`""`) [WAI-ARIA 1.1 state or property][] specified.
 
 ## Expectation
 
-Each test target has a valid value according to its [WAI-ARIA 1.1 value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value).
+Each test target has a valid value according to its [WAI-ARIA 1.1 value type][].
 
-For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState) at least one of the elements with the given ids exists in the same [document tree](https://dom.spec.whatwg.org/#document-trees) or [shadow tree](https://dom.spec.whatwg.org/#shadow-trees) as the element that specifies the target attribute.
+For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties][] at least one of the elements with the given ids exists in the same [document tree][] or [shadow tree][] as the element that specifies the target attribute.
```

Comment:
This somehow needs to mention the role of the element the property is on. Because required property depend on role.

Reply:
@Jym77 are you suggesting that we make elements that do not have a role attribute as inapplicable to this rule?

Reply:
Not sure what I was thinking about either… 🤷